apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/hentinformasjon/BrevdataFacade.kt

```diff
@@ -106,25 +116,26 @@ class BrevdataFacade(
                     }
 
                 VedtakType.TILBAKEKREVING ->
-                    (vedtak.innhold as VedtakInnholdDto.VedtakTilbakekrevingDto).let { vedtakInnhold ->
-                        GenerellBrevData(
-                            sak,
-                            personerISak,
-                            behandlingId,
-                            forenkletVedtak =
-                                ForenkletVedtak(
-                                    vedtak.id,
-                                    vedtak.status,
-                                    vedtak.type,
-                                    ansvarligEnhet,
-                                    saksbehandlerIdent,
-                                    attestantIdent,
-                                    vedtak.vedtakFattet?.tidspunkt?.toNorskLocalDate(),
-                                ),
-                            grunnlag.mapSpraak(),
-                            // tilbakekreving = TODO EY-2806
-                        )
-                    }
+                    GenerellBrevData(
+                        sak,
+                        personerISak,
+                        behandlingId,
+                        forenkletVedtak =
+                            ForenkletVedtak(
+                                vedtak.id,
+                                vedtak.status,
+                                vedtak.type,
+                                ansvarligEnhet,
+                                saksbehandlerIdent,
+                                attestantIdent,
+                                vedtak.vedtakFattet?.tidspunkt?.toNorskLocalDate(),
+                                tilbakekreving =
+                                    objectMapper.readValue(
```

Comment:
Er det nødvendig å gå via .toJson + readValue?

Reply:
Jeg fant ingen bedre måte å gjøre det på fordi readValue leser ikke objectNode og toString bruker en egen ObjectMapper og da får jeg ikke riktig format på bigdecimal verdiene.
Men har du et bedre forslag er jeg åpen for det :)

Reply:
Ah, den biten der tilbakekreving er ObjectNode gjer det krevjande, ja. Trur ikkje eg har anna forslag utan å skrive om den biten.